clients/customer-portal-client/src/openapi.d.ts

```diff
@@ -241,6 +241,87 @@ declare namespace Components {
              */
             portal_id: string;
         }
+        export interface Balance {
```

Comment:
Where these gone in some point as they want to come back now?

Reply:
This is weird, I don't remember these Billing API types being gone for them to come back 🤔

Reply:
Looks like previous version from Viljami removed them https://github.com/epilot-dev/sdk-js/pull/113
Mine brings them back now